Observer/src/observer/Subscriber.java

```diff
@@ -0,0 +1,22 @@
+package observer;
+
+public class Subscriber {
```

Comment:
There is no abstraction for subscriber
SOLI **D**

Reply:
Done